src/mediapipe_internal/mediapipegraphexecutor.cpp

```diff
@@ -812,6 +808,13 @@ Status MediapipeGraphExecutor::infer(const KFSRequest* requestPtr, KFSResponse*
         }
         SPDLOG_TRACE("Received all: {} packets for: {}", receivedOutputs, outputStreamName);
     }
+    absStatus = graph.WaitUntilIdle();
+    for (auto& inputName : this->inputNames) {
```

Comment:
There is no need to close all input streams manually one by one, just use graph.CloseAllSourcePackets